pkgs/development/python-modules/nox/default.nix

```diff
@@ -13,6 +13,7 @@
   tox,
   uv,
   virtualenv,
+  fetchpatch,
```

Comment:
unused ?

Reply:
that is true, as i was trying before to actually fetchpatch, ill give it a short force push